targets/csharp/source/source/PlayFabGameserverModels.cs

```diff
@@ -0,0 +1,182 @@
+﻿// GSDK Only has the following supported frameworks for now
+#if NETSTANDARD2_0 || NETCOREAPP2_1
+using System;
+using System.Collections.Generic;
+using System.IO;
+using PlayFab.Json;
+
+namespace PlayFab
+{
+    public class GameserverConnectedPlayer
+    {
+        public string PlayerId { get; set; }
+
+        public GameserverConnectedPlayer(string playerid)
+        {
+            this.PlayerId = playerid;
+        }
+    }
+
+    public class GSDKInitializationException : Exception
+    {
+        public GSDKInitializationException(string message)
+            : base(message) { }
+
+        public GSDKInitializationException(string message, Exception innerException)
+            : base(message, innerException) { }
+    }
+
+    class GameserverConfiguration
+    {
+        public string HeartbeatEndpoint { get; protected set; }
+        public string ServerId { get; protected set; }
+        public string LogFolder { get; protected set; }
+        public string CertificateFolder { get; set; }
+
+        /// <summary>
+        /// A folder shared by all the game servers within a VM (to cache user generated content and other data).
+        /// </summary>
+        public string SharedContentFolder { get; set; }
+
+        public IDictionary<string, string> GameCertificates { get; set; }
+        public string TitleId { get; set; }
+        public string BuildId { get; set; }
+        public string Region { get; set; }
+        public IDictionary<string, string> BuildMetadata { get; set; }
+        public IDictionary<string, string> GamePorts { get; set; }
+
+        public virtual bool ShouldLog() { return true; }
+
+        protected const string HEARTBEAT_ENDPOINT_ENV_VAR = "HEARTBEAT_ENDPOINT";
+        protected const string SERVER_ID_ENV_VAR = "SESSION_HOST_ID";
+        protected const string LOG_FOLDER_ENV_VAR = "GSDK_LOG_FOLDER";
+        protected const string TITLE_ID_ENV_VAR = "PF_TITLE_ID";
+        protected const string BUILD_ID_ENV_VAR = "PF_BUILD_ID";
+        protected const string REGION_ENV_VAR = "PF_REGION";
+        protected const string SHARED_CONTENT_FOLDER_ENV_VAR = "SHARED_CONTENT_FOLDER";
+
+        public GameserverConfiguration()
+        {
+            GameCertificates = new Dictionary<string, string>();
+            BuildMetadata = new Dictionary<string, string>();
+            GamePorts = new Dictionary<string, string>();
+        }
+    }
+
+    class EnvironmentVariableConfiguration : GameserverConfiguration
+    {
+        public EnvironmentVariableConfiguration() : base()
+        {
+            HeartbeatEndpoint = Environment.GetEnvironmentVariable(HEARTBEAT_ENDPOINT_ENV_VAR);
+            ServerId = Environment.GetEnvironmentVariable(SERVER_ID_ENV_VAR);
```

Comment:
I thought Andrew Hesky removed these from the agent?

Reply:
he did, I guess if we're not going back to environment variable config we can get rid of this ,it's not really used anymore

---
In reply to: [255769296](https://github.com/PlayFab/SDKGenerator/pull/398#discussion_r255769296) [](ancestors = 255769296)

Reply:
guess I'm getting rid of this now :P

---
In reply to: [256082785](https://github.com/PlayFab/SDKGenerator/pull/398#discussion_r256082785) [](ancestors = 256082785,255769296)